src/pages/settings/Security/CloseAccountPage.js

```diff
@@ -141,7 +144,7 @@ class CloseAccountPage extends Component {
                         </Text>
                     )}
                     onConfirm={CloseAccount.clearError}
-                    isVisible={Boolean(this.props.closeAccount.error)}
+                    isVisible={Boolean(this.props.closeAccount.error) || Boolean(this.props.closeAccount.errors)}
```

Comment:
Use `!_.isEmpty` instead. If we don't pass errors they default to an empty object which is truthy.
![image](https://user-images.githubusercontent.com/26260477/196541330-d83b23ed-4888-4c5e-ba97-7decf639f66f.png)